extends Level


var ultimatum_final = DialogNode.new("Nile frowns, a little shocked by your statement.
An ultimatum, huh? I get that this matters to you, but I donâ€™t know if I can just flip a switch like that. I donâ€™t like being told how to live my life, even if your reasons are good.", null, null, 
	func(player):
		player.hearts -= 1
		explode("ðŸ’”")
)
var ultimatum_start = DialogNode.new("$name: If you canâ€™t make that change, then I donâ€™t think I can continue this relationship", ultimatum_final)



var try_end = DialogNode.new("Nile: Iâ€¦ I get it. Iâ€™ve never really thought about this stuff like you do, but I can see how much it means to you. If itâ€™s that important to you, then yeahâ€”Iâ€™ll try. Iâ€™m not making any promises to go zero waste tomorrow, but Iâ€™ll give it a shot.", null)
var try_start = DialogNode.new("$name: I donâ€™t expect you to change everything overnight. But I need to know that youâ€™re willing to try.", try_end)

#need to make the choices
var confession_5 = Choose2.new("$name: Nile, I really like you. But thereâ€™s something I need to tell you. Sustainability isnâ€™t just something I care aboutâ€”itâ€™s a core part of who I am. I want to build something real between us, 
but... I donâ€™t think I can keep going if weâ€™re not aligned on this. I need to know that you care about this planet as much as I do.", 
try_start, "Don't demand change now", ultimatum_start, "Set an ultimatum")
var confession_4 = DialogNode.new("The sun is starting to set, painting the sky with watercolors of orange and pink. Itâ€™s the perfect time to confess.", confession_5)
var confession_3 = DialogNode.new("Nile looks at you with a soft gaze. 
Nile: I feel the same with you. Itâ€™s like everything around me disappears when Iâ€™m with you.", confession_4)
var confession_2 = DialogNode.new("$name: I really enjoy my time with you Nile, I feel so calm and free.", confession_3, NILE)
var confession_1 = DialogNode.new("You and Nile are sitting close to each other on the beach, hands almost touching. After a fun day running around the beach together, the atmosphere got tense. Itâ€™s like Nile is also anticipating a serious conversation.", confession_2)
var confession_intro = DialogNode.new("It has been a couple of dates since the cooking one. Itâ€™s enough for you to feel like itâ€™s time to confess to Nile, 
and see if heâ€™s willing to be sustainable with you. You have invited him on a beach date, with a little romantic picnic! 
Remember, tread carefully, if he doesnâ€™t choose to live sustainably after this confession, it may be impossible to ever control the water damage he can cause. ", confession_1)

var ending = false

# Called when the node enters the scene tree for the first time.
func _ready() -> void:
	self.tree = confession_1
	super._ready()

# Called every frame. 'delta' is the elapsed time since the previous frame.
func _process(delta: float) -> void:
	pass
